packages/sui-ssr/server/ssr/index.js

```diff
@@ -31,6 +32,12 @@ const injectDataHydratation = (data = {}) => {
   const escapedJson = JSON.stringify(data).replace(/<\//g, '<\\/')
   return `<script>window.__INITIAL_PROPS__=${escapedJson}</script>`
 }
+const injectDataPerformance = (
+  { getInitialProps: server, renderToString: render } = {}
+) => {
+  const escapedJson = JSON.stringify({ server, render }).replace(/<\//g, '<\\/')
```

Comment:
Are we sure we need to scape this? What are we try to escaping?

Reply:
lazy copy&paste :)

Reply:
Copypasta 🍝